### ISSUE TITLE
Consider replace/disabling IPv4 connectivity on instances

### ISSUE THREAD

**Issue by bethac07** [DistributedScience/Distributed-Something#36]
TLDR (see [this blog post](https://aws.amazon.com/blogs/networking-and-content-delivery/identify-and-optimize-public-ipv4-address-usage-on-aws/)), AWS is going to start charging for assigning IP addresses to instances. 

Looking at our own usage, in a 6 month period we would have racked up ~$500 in extra costs, which is not fatal obviously but the only reason to have addresses assigned is so you can log into the individual machine to peek inside the Docker container (or the system in general), so there is rarely if ever a need to actually do it now that we have logging (I used to do this a lot before we did have logging, but it was annoying, which is why we now have logging). So I'd argue for just disabling it, but we could make it optional, default no.

**Comment by ErinWeisbart** [comment]
Fix in DCP is https://github.com/DistributedScience/Distributed-CellProfiler/pull/182/files

**Comment by bethac07** [comment]
Should we have a fix here too?

**Comment by ErinWeisbart** [comment]
Yep, planning on it!

**Comment by ErinWeisbart** [comment]
fixed in https://github.com/DistributedScience/Distributed-Something/commit/d1d6e62edb0c83e5d0a047e8569d455ea2cebacd
